prow/spyglass/viewers/metadata/metadata_viewer.go

```diff
@@ -98,10 +98,12 @@ func ViewHandler(artifacts []viewers.Artifact, raw string) string {
 		}
 
 	}
-	d := Derived{
-		Elapsed: metadataViewData.Finished.Timestamp.Sub(metadataViewData.Started.Timestamp),
+	if metadataViewData.Finished.TimestampRaw == 0 {
+		metadataViewData.Derived.Elapsed = time.Now().Sub(metadataViewData.Started.Timestamp)
+	} else {
+		metadataViewData.Derived.Elapsed =
```

Comment:
perhaps instead we should render from a string, and put something like "(running)" at the end when we don't have a finished time?

Reply:
(or the template can handle this)